hlx_statics/components/code.js

```diff
@@ -0,0 +1,13 @@
+export default function decorateCode(block) {
+  console.log('~~ 2');
```

Comment:
Just need to remove this console log

Reply:
Oh whoop - I didn't realise this is still in draft mode. Just let me know when it's good to go!